rclcpp/include/rclcpp/graph_event.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCLCPP__GRAPH_EVENT_HPP_
+#define RCLCPP__GRAPH_EVENT_HPP_
+
+#include "rclcpp/event.hpp"
+
+
+namespace rclcpp
+{
+
+class GraphEvent : public Event {};
```

Comment:
Since you undid the GraphEvent renaming, you should remove this header file.

Reply:
Good point